packages/tools/kadena-cli/src/devnet/utils/simulation/file.ts

```diff
@@ -0,0 +1,57 @@
+import fs from 'fs';
+import path from 'path';
+import { services } from '../../../services/index.js';
+
+export interface IFileData {
+  timestamp: number;
+  from: string;
+  to: string;
+  amount: number;
+  requestKey: string;
+  action: TransferType | TokenActionType;
+}
+
+export interface IFileError {
+  error: string;
+}
+
+export type TransferType =
+  | 'fund'
+  | 'transfer'
+  | 'cross-chain-transfer'
+  | 'safe-transfer'
+  | undefined;
+
+export type TokenActionType = 'create' | 'mint' | 'transfer' | undefined;
+
+export function createDir(directory: string): void {
```

Comment:
move to filesystem service, so others can use this also, (if not already there)

Reply:
there is a `ensureDirectoryExists` method

Reply:
Fixed